lib/logger.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var exists = require('101/exists');
+var bunyan = require('bunyan');
+var Bunyan2Loggly = require('bunyan-loggly').Bunyan2Loggly;
```

Comment:
this had memory leaks, we are removing bunyan2loggly in favor of just writing to stdout.